pkg/policy/repository.go

```diff
@@ -38,12 +39,28 @@ type Repository struct {
 	// incremented whenever the policy repository is changed.
 	// Always positive (>0).
 	revision uint64
+
+	// RepositoryChangeQueue is a queue which serializes changes to the policy
+	// repository.
+	RepositoryChangeQueue *eventqueue.EventQueue
+
+	// RuleReactionQueue is a queue which serializes the resultant events that
+	// need to occur after updating the state of the policy repository. This
+	// can include queueing endpoint regenerations, policy revision increments
+	// for endpoints, etc.
+	RuleReactionQueue *eventqueue.EventQueue
```

Comment:
nit: Strictly speaking this part is no-op right now so should go into the other PR, but I expect at this point that both PRs should go in anyway (modulo review) so it's not that big of a deal to have it here.